components/Star/index.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+export default function Star({ onRate, full,onHoverIn,onHoverOut }) {
+  return (
+    <MainStarDiv
+      role="button"
+      onClick={onRate}
+      onMouseEnter={onHoverIn}
+      onMouseLeave={onHoverOut}
+    >
+      {full ? (
+        <svg
```

Comment:
That's not that lovely code 😀. It makes sense to outsource the svg's and import them.